src/Form.tsx

```diff
@@ -1,10 +1,26 @@
-import React, { useState } from "react";
+import { useState } from "react";
 import "./Form.css";
 
-const Form = ({ onFormSubmit, games, languages }) => {
-  const [visible, setVisible] = useState(false);
-  const [campaign, setCampaign] = useState({
-    name: "",
+interface Campaign {
```

Comment:
You have one here and one src/campaign.ts which is right one?

Reply:
Okey, now I import this from src/interfaces/campaign.ts in Form.tsx.